### PR TITLE
Add Finnish PO file (CLI)

### LINKED ISSUE #191: Add Finnish translation in CLI
1. Add `fi.po` for the text the CLI outputs, except the test messages that comes from Zonemaster-Engine. (#192)
2. Update installation instruction to add fi_FI where needed (#193)

This issue assumes that https://github.com/zonemaster/zonemaster-engine/issues/879 is resolved.

### PR DESCRIPTION
**Pull request by matsduf**
This PO file has been sent by email by translator. It needs some corrections before it can be merged. This resolves #191, part 1.

### CODE REVIEW

File: share/fi.po [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+"this system?).\n"
+"\n"
+msgstr ""
+"Varoitus: arvoa %s ei voitu asettaa lokaalikategorian LC_CTYPE arvoksi (onko "
+"sitä asennettu tähän järjestelmään?).\n"
+"\n"
+
+#, perl-brace-format
+msgid "Address {address} cannot be used as source address for DNS queries.\n"
+msgstr ""
```
Comment (matsduf) [inline share/fi.po:193]: Untranslated. Needs to be fixed by translator.

**Comment by matsduf** [comment]
@samisalmensuo, can you look at this?

File: share/fi.po (hunk 1 above)
Comment (samisalmensuo) [reply share/fi.po:193]: "Osoitetta {address} ei voi käyttää lähdeosoitteena DNS kyselyille.\n"

Comment (matsduf) [reply share/fi.po]: Thanks. Included.

**Review by mattias-p — APPROVED** [review]

**Comment by matsduf** [comment]
To test this PR with support for Finnish, follow the [installation instructions] with the following modifications:

1. If installing on Ubuntu or Debian also include `fi_FI.UTF-8` when updating the locale instructions.
2. Fetch [Zonemaster-CLI-v3.0.4-PR-192-ba22489-2021-03-17.tar.gz]
3. Instead of the command `cpanm Zonemaster::CLI`, run
```
cpanm Zonemaster-CLI-v3.0.4-PR-192-ba22489-2021-03-17.tar.gz
```
4. Test with the following command that should give messages in Finnish:
```
zonemaster-cli --locale fi_FI.UTF-8 --level INFO zonemaster.net
```

When Zonemaster-CLI is installed, go to zonemaster/zonemaster-backend#712 to install a test version of Zonemaster-Backend with support for Finnish.

[Zonemaster-CLI-v3.0.4-PR-192-ba22489-2021-03-17.tar.gz]: https://github.com/zonemaster/zonemaster-cli/files/6157933/Zonemaster-CLI-v3.0.4-PR-192-ba22489-2021-03-17.tar.gz
[installation instructions]: https://github.com/zonemaster/zonemaster-cli/blob/master/docs/Installation.md

**Comment by pnax** [comment]
## Release testing for v2021.1

I've verified this test procedure on Debian 10, Ubuntu 18.04 and Ubuntu 20.04.
I've also checked that with `--level DEBUG` the messages are in Finnish (at least not in English for some of them)

**Pull request merged**